src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-card/sw-customer-card.html.twig

```diff
@@ -22,6 +22,12 @@
                                                     <template v-if="!editMode">
                                                         <div v-if="customer && customer.salutation" class="sw-customer-card__metadata-customer-name">
                                                             {{ fullName }}
+                                                            <sw-label v-if="customer.guest"
+                                                                             appearance="pill"
```

Comment:
Sorry my fault, could you please line up the `appearance` , `size` and class with the `v-if` attribute.

It should look like this:
```
<sw-label v-if="!isInlineEdit && item.guest"
          appearance="pill"
          size="small"
          class="sw-customer-list__metadata-customer-guest-label">
```

The suggest tool doesn't work like expected.

Reply:
sure